recipes/recipes_emscripten/simplejson/recipe.yaml

```diff
@@ -0,0 +1,13 @@
+context:
+  version: 3.19.1
+
+package:
+  name: simplejson
+  version: "{{ version }}"
+
+build:
+  number: 0
```

Comment:
since there are some C part to this library:
https://github.com/simplejson/simplejson/blob/master/simplejson/_speedups.c
shouldn't we also add `{{ compiler('c') }}` as a build requirement (see https://github.com/conda-forge/simplejson-feedstock/blob/main/recipe/meta.yaml#LL23C7-L23C26) or disable the speed ups with `REQUIRE_SPEEDUPS` to make it pure python.

Reply:
@proppy absolutely correct!